CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
     - `search` to search for audit-logs.
     - `send-to` to send audit-logs to server.
 
+
```

Comment:
Looks like our changelog has gotten messed up? The ` #Unreleased` tag should not be there...

Reply:
@unparalleled-js I re-pulled from master. The #Unreleased tag should be at the top, right? Let me know if I should fix it with this PR